src/pages/auth/__tests__/SignIn.test.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { renderWithProvider } from '../../../utils/testing';
+import { MemoryRouter, Route } from 'react-router-dom';
+import SignIn from '../SignIn';
+import mascotapi from '../../../api/mascotapi';
+import { act, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('../../../api/mascotapi', () => ({
+    post: jest.fn()
+}));
+
+describe('SignIn', () => {
+    let testLocation;
+    beforeEach(() => {
+        renderWithProvider(
+            <MemoryRouter initialEntries={['/signin']}>
+                <SignIn />
+                <Route
+                    path="*"
+                    render={({ location }) => {
+                        testLocation = location;
+                        return null;
+                    }}
+                />
+            </MemoryRouter>
+        );
+        mascotapi.post.mockImplementation(() => Promise.resolve());
+    });
+
+
+    it("renders correctly", () => {
+        expect(screen.getByText('signIn.title')).toBeInTheDocument();
+        expect(screen.getByText('signIn.labels.email')).toBeInTheDocument();
+        expect(screen.getByText('signIn.labels.password')).toBeInTheDocument();
+        expect(screen.getByText('signIn.actions.signIn')).toBeInTheDocument();
+    });
+
+    describe('when user set valid data', () => {
+        const token = 'fakeToken';
+        const emailValue = 'email@domain.cl';
+        const passwordValue = 'awesome-password';
+
+        beforeEach(() =>
+            mascotapi.post.mockImplementation(
+                () =>
+                    new Promise(resolve => {
+                        resolve({
+                            headers: { Authorization: token },
+                            status: 200,
+                            data: { email: emailValue, password: passwordValue }
+                        });
+                    })
+            )
+        );
+
+        it('login the user correctly', async () => {
```

Comment:
signs the user in* es una cosa contextual entre login y sign in. login se refiere a que se logea  la actividad parcial o total del usuario, sin embargo nosotros no hacemos eso